### ISSUE TITLE
A4 Wide Monster Stat Block is Single Column

### ISSUE THREAD

**Issue by gwilym** [stolksdorf/naturalcrit#102]
**Browser Type/Version**: Google Chrome Version 50.0.2661.102 (64-bit)

**Operating System**: OS X El Capitan 10.11.4 (15E65)

**Issue Description**: Using the A4 snippet with a wide monster stat block renders the stat block in a single column. Expected two columns.

**Markdown code to reproduce**:

```
<style>
  .phb{
    width : 210mm;
    height : 296.8mm;
  }
</style>
___
___
> ## Mad Koala of the West
>*Large cutie, lawful coy*
> ___
> - **Armor Class** 14
> - **Hit Points** 117(1d4 + 5)
> - **Speed** 4ft.
>___
>|STR|DEX|CON|INT|WIS|CHA|
>|:---:|:---:|:---:|:---:|:---:|:---:|
>|4 (-3)|17 (+4)|18 (+4)|5 (-2)|16 (+3)|7 (-1)|
>___
> - **Condition Immunities** weak-kneed, melancholy, buzzed
> - **Senses** passive Perception 16
> - **Languages** None
> - **Challenge** 3 (4796 XP)
> ___
> ***Pack Tactics.*** These guys work together. Like super well, you don't even know.
>
> ***False Appearance. *** While the armor reamin motionless, it is indistinguishable from a normal suit of armor.
>
> ***Pack Tactics.*** These guys work together. Like super well, you don't even know.
> ### Actions
> ***Bulldog Rake.*** *Melee Weapon Attack:* +4 to hit, reach 5ft., one target. *Hit* 5 (1d6 + 2) 
>
> ***Jumping Driver.*** *Melee Weapon Attack:* +4 to hit, reach 5ft., one target. *Hit* 5 (1d6 + 2) 
>
> ***Tilt-a-whirl Chin Sleeper.*** *Melee Weapon Attack:* +4 to hit, reach 5ft., one target. *Hit* 5 (1d6 + 2) 
>
> ***Team Elbow.*** *Melee Weapon Attack:* +4 to hit, reach 5ft., one target. *Hit* 5 (1d6 + 2) 
>
> ***Crossed Splash.*** *Melee Weapon Attack:* +4 to hit, reach 5ft., one target. *Hit* 5 (1d6 + 2) 
```

Not-thoroughly-tested workaround for me (I'm not a modern CSS expert) is to override the column with:

```
  .phb hr+hr+blockquote{
    column-width: 7.7cm;
  }
```

**Related Images** :
![image](https://cloud.githubusercontent.com/assets/270947/15459790/87e70504-2098-11e6-8467-fc32db2e4cb1.png)

**Comment by stolksdorf** [comment]
I see what happened. The extra spacing due to the new image-border squeezed the contents too much. The `7.7cm` fix was on point, but also needed to be a pplied in a few other places. Good catch!

The fix is up on [staging](natural-crit-staging.herokuapp.com/homebrew), can you verify it's working and test it alittle? Once you confirm it's fixed I'll deploy it to prod.

Thanks for the heads up!

**Comment by stolksdorf** [comment]
Fixed in v2.0.4

**Comment by gwilym** [comment]
LGTM on staging, but I didn't test extensively, just basically pasted in my example from above. Thanks 👍
